Detectors/FIT/base/include/FITBase/Hit.h

```diff
@@ -0,0 +1,46 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_FIT_HIT_H
+#define ALICEO2_FIT_HIT_H
+
+#include "SimulationDataFormat/BaseHits.h"
+
+namespace o2 {
+  namespace FIT {
```

Comment:
the official namespace naming convention (see https://github.com/AliceO2Group/CodingGuidelines) say that namespaces should be lowercase. We will impose this soon. Since this is new code, I suggest to already follow this rule now.